app/assets/locales/locale-de.json

```diff
@@ -957,6 +957,7 @@
         "scan_qr_code": "Scan QR code"
     },
     "header": {
+        "accounts_manage": "Konten verwalten",
```

Comment:
Does this need to be duplicated like this?

Reply:
It's two different translates. First one is default translate for icon and the second one is for dropdown when user click on his username.